proto/xudrpc.proto

```diff
@@ -306,6 +306,8 @@ message SwapFailure {
   string peer_pub_key = 4 [json_name = "peer_pub_key"];
   // The reason why the swap failed.
   string failure_reason = 5 [json_name = "failure_reason"];
+  // The fail time for the swap.
```

Comment:
I might say "The epoch time in milliseconds when the swap failed" to be a little more clear.